src/citrine/_utils/functions.py

```diff
@@ -5,10 +5,13 @@
 from urllib.parse import quote, urlencode, urlparse
 from uuid import UUID
 from warnings import warn
+import warnings
+warnings.filterwarnings('always', category=DeprecationWarning)
```

Comment:
I think this is just here for testing, but just flagging to ensure we remove it before merge.

Reply:
Yep! Just testing. They weren't showing up in my notebook otherwise for some reason